EasyPost/Parameters/V2/CarrierAccounts.cs

```diff
@@ -0,0 +1,209 @@
+using System.Collections.Generic;
+using EasyPost.Utilities.Annotations;
+
+namespace EasyPost.Parameters.V2
+{
+    public static class CarrierAccounts
+    {
+        /// <summary>
+        ///     Parameters for <see cref="EasyPost.Services.CarrierAccountService.Create"/> creation API calls.
+        /// </summary>
+        public sealed class Create : CreateRequestParameters
+        {
+            #region Request Parameters
+
+            [RequestParameter(Necessity.Optional, "carrier_account", "credentials")]
+            public Dictionary<string, object?>? Credentials { get; set; }
+
+            [RequestParameter(Necessity.Optional, "carrier_account", "description")]
+            public string? Description { get; set; }
+
+            [RequestParameter(Necessity.Optional, "carrier_account", "reference")]
+            public string? Reference { get; set; }
+
+            [RequestParameter(Necessity.Optional, "carrier_account", "test_credentials")]
+
+            // ReSharper disable once InconsistentNaming
+            public Dictionary<string, object?>? TestCredentials { get; set; }
+
+            [RequestParameter(Necessity.Optional, "carrier_account", "type")]
+            public string? Type { get; set; }
+
+            #endregion
+        }
+
+        /// <summary>
+        ///     Parameters for FedEx <see cref="EasyPost.Services.CarrierAccountService.Create"/> creation API calls.
+        /// </summary>
+        public sealed class CreateFedEx : CreateRequestParameters
```

Comment:
### Author's Note

We can get rid of these FedEx- and UPS-specific parameter sets if we don't want to worry about that for now.